about-us.html

```diff
@@ -127,7 +127,7 @@ <h2>GOALS</h2>
                     <div class="col-md-4">
                         <div class="border-box boxDownload2">
                             <h2 >Education Goals</h2>
-                            <p>Sugar is useful only to the extent it is used by the learning community. Thus, Sugar Labs is working with educators around the world to focus on these learning challenges:</p>
+                            <p>Sugar is useful only to the extent it is used by the learning community. Thus, Sugar Labs is working with educators around the world to focus on these learning challenges:<pre></pre></p>
```

Comment:
What's this `<pre></pre>` added for?

Reply:
`<pre></pre>` is used here for line breaks without new text. So that the 1st and 3rd blocks look similar and inline, because of the flex the middle block is extended to keep things looking similar, I used it.
_Earlier:_
![WhatsApp Image 2022-01-13 at 8 44 50 AM](https://user-images.githubusercontent.com/86892991/149260124-d1bfd9e2-6dcc-42e5-8d65-db64efd3d053.jpeg)
_Now:_
![WhatsApp Image 2022-01-13 at 8 45 44 AM](https://user-images.githubusercontent.com/86892991/149260167-d94abce3-d4eb-4d8a-8c29-8764b62bf92d.jpeg)